CMakeLists.txt

```diff
@@ -125,23 +125,33 @@ if(NOT SERVER_DIR)
     endif()
 endif()
 
-include(ExternalProject)
-set(UDP_NOTIF_BIN ${CMAKE_CURRENT_BINARY_DIR}/lib)
-set(UDP_NOTIF_STATIC_LIB ${UDP_NOTIF_BIN}/lib/libunyte-udp-notif.a)
-set(UDP_NOTIF_INCLUDES ${UDP_NOTIF_BIN}/include)
-
-ExternalProject_Add(udp-notif-c-collector
-    PREFIX ${UDP_NOTIF_BIN}
-    GIT_REPOSITORY https://github.com/network-analytics/udp-notif-c-collector
-    GIT_TAG main
-    BUILD_IN_SOURCE 1
-    UPDATE_DISCONNECTED true
-    PATCH_COMMAND patch -p1 < ${CMAKE_CURRENT_SOURCE_DIR}/patches/0001-feat-add-local-address-to-sender.patch
-    CONFIGURE_COMMAND ./bootstrap && ./configure --prefix=${UDP_NOTIF_BIN}
-    BUILD_COMMAND make
-    INSTALL_COMMAND make install
-    BUILD_BYPRODUCTS ${UDP_NOTIF_STATIC_LIB}
-)
+if (NOT UDP_NOTIF_BIN)
+    set(BUILD_UDP_NOTIF yes)
```

Comment:
I do not really understand the logic of these variables, are they even required? Is this feature supposed to be optional? If so, the variable is supposed to be with the options at the beginning. In any case, please simplify this and do not use redundant variables. If you are trying to implement caching of this external project (not sure if needed, I think `cmake` can handle that on its own), using `UDP_NOTIF_STATIC_LIB` and `UDP_NOTIF_INCLUDES` should be enough.